core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -56,6 +61,13 @@ <h1 class="stat-value-without-rating" ng-hide="dashboardStats.num_ratings || rel
         </div>
       </md-card>
     </div>
+    <div class="row">
```

Comment:
Is all this new functionality supposed to go fully live in the upcoming release? If so, I'd be surprised, since Questions functionality is not generally released yet.

If it's not supposed to go fully live yet, shouldn't this stuff be behind an ng-if condition?

Reply:
No this isn't ready for release (Will get released along with the new structures project I guess). Yep, I missed adding a flag, will do so. Shall I create a new flag, (as otherwise it will be a combination of ENABLE_GENERALIZED_FEEDBACK_THREADS and ENABLE_NEW_STRUCTURES).

Reply:
Oh dear. That's a major omission, glad it got caught.

Don't create a new flag. Use a combination of both existing flags.

Reply:
@nithusha21 ping?

Reply:
Done